src/IO/WriteBufferFromS3MemoryStream.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "config.h"
+
+#if USE_AWS_S3
+
+#include <iostream>
+
+namespace DB
+{
+
```

Comment:
Why is the file is named `WriteBufferFromS3MemoryStream`? And please add a comment here for this class.